use-cases/org-existing-cloudtrail-ecs-vpc-subnet.md

```diff
@@ -1,41 +1,35 @@
-# OrganizationSetup - Existing Cloudtrail - Existing ECS/VPC/Subnet
+# AWS Organizational Setup with Cloudtrail
```

Comment:
we will always have a cloudtrail.
will change this in further PR to address the important part of the use-case, which is the ECS/VPC/Subnet usage